test/unit/test-mysqlDaoQueryHelper.js

```diff
@@ -103,6 +104,36 @@ describe('MysqlDaoQueryHelper', function() {
             sql.should.eql("SELECT * FROM users WHERE (first_name IS NULL) AND (last_name = 'Doe') LIMIT 1");
         });
 
+        it('Should generate dates with for a utc timezone ', function() {
+            const now = moment();
+            const nowDate = now.toDate();
+            const utcDateString = now.format('YYYY-MM-DD HH:mm:ss.SSS');
+
+            const sql = mysqlDaoQueryHelper.getOne({dateAdded: nowDate});
+            Should.exist(sql);
+
+            sql.should.eql(`SELECT * FROM users WHERE (date_added = '${utcDateString}') LIMIT 1`);
+        });
+
+        before(() => {
+            mysqlDaoQueryHelper._clean = value => Mysql.escape(value);
```

Comment:
Instead of before/after, I would probably create two different helpers, one with a local timezone clean function and one with a UTC clean function.  Should be straight forward b/c the clean function is passed in via the constructor.